app/src/main/java/chat/rocket/android/service/RocketChatWebSocketThread.java

```diff
@@ -238,6 +252,35 @@ public void run() {
                   return null;
                 });
 
+                task.getResult().client.getOnFailureCallback().onSuccess(_task -> {
+                  ddpClient = null;
+                  CompositeDisposable subscriptions = new CompositeDisposable();
```

Comment:
I'll change those :) Just kept this way to keep the same naming convention already being used